ll.tm

```diff
@@ -807,6 +807,7 @@ TopLevelEntity -> TopLevelEntity
 	| MetadataDef
 	| UseListOrder
 	| UseListOrderBB
+	| Preallocated
```

Comment:
The `preallocated` attribute is only present as a function attribute or a parameter attribute, never as a top-level entity. This can be verified by checking where invocations to `ParsePreallocated` are made; namely from `ParseFnAttributeValuePairs` and `ParseOptionalParamAttrs`.

This we should remove this line.
```suggestion
```

Reply:
You're right XD